src/executable/DprintExecutable.ts

```diff
@@ -170,7 +170,7 @@ async function tryResolveNpmExecutable(cwd: vscode.Uri) {
 
   try {
     await vscode.workspace.fs.stat(npmExecutablePath);
-    return npmExecutablePath.fsPath;
+    return `"${npmExecutablePath.fsPath}"`;
```

Comment:
I tried that, but it didn't work because `spawn` also needs the quotes:

https://github.com/dprint/dprint-vscode/blob/4ae6c6874c3f3040423fccb02f7f49971f62dd01/src/executable/DprintExecutable.ts#L110

So we either add it to the `spawn` as well:

```js
return spawn(`"${this.#cmdPath}"`, args, { 
```

or we can add it to the `cmdPath` getter and use `this.cmdPath` (instead of `this.#cmdPath`) inside the class.

I'm going to push that version because it looks more robust to me, but let me know what you think.

Reply:
Yeah, we should update `spawn` as well in that case to keep the shell concerns altogether. `cmdPath` shouldn't be concerned with needing to add quotes because someone developing should be able to use that value with non-shell commands as well.